compilation under windows fails with `1.3.2`

The final error message is: `  = note: LINK : fatal error LNK1181: cannot open input file 'matio.lib'`.

I guess this is likely related to a windows percularity: `target\debug\build\matio-rs-sys-*\out\build\Debug\libmatio.lib` does exist, but the corresponding `build.rs` refers to `matio`. While linkers on Linux add prefixes like `lib` and suffixes like `.a`, windows' `link.exe` does not do automatically add them. Hence the linker looks for `matio.lib`, but the actual file is called `libmatio.lib`.

https://github.com/rconan/matio-rs/blob/e58fffe3d29be798eed463096791b64f6b592642/sys/build.rs#L16

--- a/README.md
+++ b/README.md
@@ -15,3 +15,13 @@ MATLAB MAT file I/O C library
 sudo apt install cmake zlib1g-dev
 ```
 
+## Installing dependencies on Windows
+
+Download and install [LLVM 15.0.7](https://github.com/llvm/llvm-project/releases/tag/llvmorg-15.0.7)
+
+```powershell
+winget install cmake
+vcpkg install zlib
+```
+
+Set environment variable ZLIB_LIB_DIR to `<vcpkg>\packages\zlib_x64-windows\lib`

--- a/sys/build.rs
+++ b/sys/build.rs
@@ -13,8 +13,20 @@ fn main() {
         "cargo:rustc-link-search={}",
         dbg!(out.join("lib64").display())
     );
-    println!("cargo:rustc-link-lib=matio");
-    println!("cargo:rustc-link-lib=z");
+
+    if cfg!(target_os = "windows") {
+        println!("cargo:rustc-link-lib=libmatio");
+        if let Ok(lib_dir) = std::env::var("ZLIB_LIB_DIR") {
+            println!("cargo:rustc-link-search={}", lib_dir);
+        } else {
+            panic!("ZLIB_LIB_DIR environment variable not set");
+        }
+        println!("cargo:rustc-link-lib=zlib");
+    } else {
+        println!("cargo:rustc-link-lib=matio");
+        println!("cargo:rustc-link-lib=z");
+    }
+
     println!("cargo:rerun-if-changed=wrapper.h");
 
     let bindings = bindgen::Builder::default()

